R/print.R

```diff
@@ -442,6 +442,66 @@ print.bbi_nmboot_summary <- function(x, .digits = 3, .nrow = 10, ...) {
   }
 }
 
+
+#' @describeIn print_bbi Prints the `NM-TRAN` evaluation of a `bbi_nonmem_model`
+#'  object
+#' @export
+print.nmtran_process <- function(x, ...){
+
+  is_valid_print <- function(.x) {
+    if (!is.null(.x)) {
+      length(.x) != 0
+    } else {
+      FALSE
+    }
```

Comment:
[ I realize this is taken from `print.bbi_model`. I'm okay leaving it as is. ]

This entire function body can be reduced to the second line, `length(.x) != 0`.

At that point, there's not much value in the `is_valid_print` helper, and, to my eyes, it'd be clearer for all the downstream conditions to say `if (length(x[[...]]))` instead of `if (is_valid_print(x[[...]]))`.

Reply:
Going to leave as-is for now for consistency

Reply:
sin bets sin :)